Source/Frontend/UI/Components/Memory Tools/Common.cs

```diff
@@ -0,0 +1,27 @@
+namespace RTCV.UI
+{
+    using System.IO;
+    using System.Windows.Forms;
+
+    internal static class Common
+    {
+        internal static bool CopyFileWithOverwritePrompt(string sourcePath, string targetDirectory)
+        {
+            var shortPath = sourcePath.Substring(sourcePath.LastIndexOf('\\') + 1);
+            var targetPath = Path.Combine(targetDirectory, shortPath);
+
+            if (File.Exists(targetPath))
+            {
+                var result = MessageBox.Show("This file already exist in your VMDs folder, do you want to overwrite it?", "Overwrite file?", MessageBoxButtons.YesNo, MessageBoxIcon.Warning);
```

Comment:
`"...in your VMDs folder..."` won't always be correct, since sometimes it's `RtcCore.listsDir`, but I left it because of the previous functionality.